Sources/KlaviyoUI/KlaviyoWebView/Resources/Scripts/consoleHandler.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * WARNING This JS file should be ES5 compatible to support older versions of Android WebView
+ *
```

Comment:
```suggestion
 * WARNING This JS file should be ES5 compatible to support older versions of Android WebView
 *
```
This was mostly an android issue, may as well remove